.github/workflows/ci.yml

```diff
@@ -331,7 +329,7 @@ jobs:
             /home/runner/.gsutil/tracker-files \
             /home/runner/.gsutil/tracker-files/upload_TRACKER_*.rc.zip__JSON.url \
           || true
-          gsutil -m cp -L gsutil.log ./libs.tar ./relay-debug.zip ./relay.src.zip ./release-name \
+          gsutil -m cp -L gsutil.log ./${{ matrix.image_name }}-libs.tar ./${{ matrix.image_name }}-debug.zip ./${{ matrix.image_name }}.src.zip ./${{ matrix.image_name }}-release-name \
```

Comment:
Idea: In devinfra assets, what if instead of renaming every file, we just put them in separate folders, such as: `.../$REVISION/relay/` and `.../$REVISION/relay-internal/`?